!SESSION 2021-05-05 22:37:56.742 -----------------------------------------------
eclipse.buildId=unknown
java.version=11.0.5
java.vendor=Amazon.com Inc.
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

This is a continuation of log file D:\year4-semester2\Silicon_lab_year3\.metadata\.bak_0.log
Created Time: 2021-05-05 22:38:28.895

!ENTRY org.eclipse.equinox.p2.transport.ecf 2 0 2021-05-05 22:38:28.896
!MESSAGE Connection to https://updates.silabs.com/studio/v5/updates/p2.index failed on updates.silabs.com. Retry attempt 0 started
!STACK 0
java.net.UnknownHostException: updates.silabs.com
	at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1505)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1364)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1298)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:112)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.eclipse.ecf.provider.filetransfer.httpclient45.HttpClientRetrieveFileTransfer.performConnect(HttpClientRetrieveFileTransfer.java:1000)
	at org.eclipse.ecf.provider.filetransfer.httpclient45.HttpClientRetrieveFileTransfer.access$0(HttpClientRetrieveFileTransfer.java:992)
	at org.eclipse.ecf.provider.filetransfer.httpclient45.HttpClientRetrieveFileTransfer$1.performFileTransfer(HttpClientRetrieveFileTransfer.java:988)
	at org.eclipse.ecf.filetransfer.FileTransferJob.run(FileTransferJob.java:74)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY com.silabs.ss.tool.installer.core 4 0 2021-05-05 22:38:29.120
!MESSAGE Unknown Host: https://updates.silabs.com/studio/v5/updates/content.xml
!STACK 1
org.eclipse.equinox.p2.core.ProvisionException: Unknown Host: https://updates.silabs.com/studio/v5/updates/content.xml
	at org.eclipse.equinox.internal.p2.repository.CacheManager.createCache(CacheManager.java:247)
	at org.eclipse.equinox.internal.p2.metadata.repository.SimpleMetadataRepositoryFactory.getLocalFile(SimpleMetadataRepositoryFactory.java:69)
	at org.eclipse.equinox.internal.p2.metadata.repository.SimpleMetadataRepositoryFactory.load(SimpleMetadataRepositoryFactory.java:89)
	at org.eclipse.equinox.internal.p2.metadata.repository.MetadataRepositoryManager.factoryLoad(MetadataRepositoryManager.java:63)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.loadRepository(AbstractRepositoryManager.java:775)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.loadRepository(AbstractRepositoryManager.java:676)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.basicRefreshRepository(AbstractRepositoryManager.java:204)
	at org.eclipse.equinox.internal.p2.metadata.repository.MetadataRepositoryManager.refreshRepository(MetadataRepositoryManager.java:115)
	at com.silabs.ss.tool.internal.api.installer.core.SLSP2UpdateSites.ensureStudioReposLoaded(SLSP2UpdateSites.java:666)
	at com.silabs.ss.tool.internal.api.installer.core.InstallerUtils.ensurePackagesFetched(InstallerUtils.java:1934)
	at com.silabs.ss.tool.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:215)
	at com.silabs.ss.tool.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:286)
	at com.silabs.ss.tool.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.net.UnknownHostException: updates.silabs.com
	at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1505)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1364)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1298)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:112)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.eclipse.ecf.provider.filetransfer.httpclient45.HttpClientFileSystemBrowser.runRequest(HttpClientFileSystemBrowser.java:246)
	at org.eclipse.ecf.provider.filetransfer.browse.AbstractFileSystemBrowser$DirectoryJob.run(AbstractFileSystemBrowser.java:69)
	... 1 more
!SUBENTRY 1 org.eclipse.equinox.p2.transport.ecf 4 1006 2021-05-05 22:38:29.121
!MESSAGE Unknown Host: https://updates.silabs.com/studio/v5/updates/content.xml
!STACK 0
java.net.UnknownHostException: updates.silabs.com
	at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1505)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1364)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1298)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:112)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.eclipse.ecf.provider.filetransfer.httpclient45.HttpClientFileSystemBrowser.runRequest(HttpClientFileSystemBrowser.java:246)
	at org.eclipse.ecf.provider.filetransfer.browse.AbstractFileSystemBrowser$DirectoryJob.run(AbstractFileSystemBrowser.java:69)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY com.silabs.ss.tool.installer.core 2 0 2021-05-05 22:38:29.182
!MESSAGE Problem getting package resources: icon

!ENTRY com.silabs.ss.platform.content.core 4 0 2021-05-05 22:38:29.194
!MESSAGE Failed to connect, verify on line, due to exception https://updates.silabs.com/studio/v5/assets/
!STACK 0
java.net.UnknownHostException: updates.silabs.com
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:220)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
	at java.base/java.net.Socket.connect(Socket.java:609)
	at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:285)
	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:177)
	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:474)
	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:569)
	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:265)
	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:372)
	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1081)
	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:168)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:1088)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.checkForUpdates(AssetService.java:542)
	at com.silabs.ss.tool.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:237)
	at com.silabs.ss.tool.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:286)
	at com.silabs.ss.tool.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY com.silabs.ss.platform.content.core 4 0 2021-05-05 22:38:29.198
!MESSAGE Failed to connect, verify on line, due to exception http://updates.silabs.com/studio/v5/assets/
!STACK 0
java.net.UnknownHostException: updates.silabs.com
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:220)
	at java.base/java.net.Socket.connect(Socket.java:609)
	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:177)
	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:474)
	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:569)
	at java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:242)
	at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:341)
	at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:362)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1248)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1081)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1015)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:1103)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.checkForUpdates(AssetService.java:542)
	at com.silabs.ss.tool.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:237)
	at com.silabs.ss.tool.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:286)
	at com.silabs.ss.tool.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY com.silabs.ss.platform.content.core 2 0 2021-05-05 22:38:29.199
!MESSAGE Assets checkForUpdates failed due to not online

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:39:07.891
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:45:22.049
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:46:27.672
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:46:45.159
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:47:43.378
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:48:20.747
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:48:38.390
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:49:21.372
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:49:34.238
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:51:21.161
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:53:09.459
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:54:53.393
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:55:18.754
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 22:58:38.930
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 23:02:00.316
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-05 23:04:16.842
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.platform.rcp.core 1 0 2021-05-05 23:14:15.421
!MESSAGE Deleted Shared Daemon File: C:\Users\sengt\.uc\cli\mainstudio.running

!ENTRY com.silabs.ss.platform.rcp.core 1 0 2021-05-05 23:14:15.422
!MESSAGE Deleted Shared Daemon Version File: C:\Users\sengt\.uc\cli\mainstudio.running.version

!ENTRY com.silabs.ss.framework.debugger.tcf.core 4 0 2021-05-05 23:14:15.491
!MESSAGE TCF task failed
!STACK 0
java.io.IOException: Command [C 1000 DebugAdapter disconnect "root.000050120677"] aborted
	at org.eclipse.tcf.core.AbstractChannel$9.run(AbstractChannel.java:838)
	at org.eclipse.tcf.EventQueue.run(EventQueue.java:95)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/java.io.BufferedInputStream.read1(BufferedInputStream.java:290)
	at java.base/java.io.BufferedInputStream.read(BufferedInputStream.java:351)
	at java.base/java.io.FilterInputStream.read(FilterInputStream.java:107)
	at org.eclipse.tcf.core.ChannelTCP.get(ChannelTCP.java:185)
	at org.eclipse.tcf.core.StreamChannel.read(StreamChannel.java:84)
	at org.eclipse.tcf.core.AbstractChannel$1.run(AbstractChannel.java:327)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 4 0 2021-05-05 23:14:15.492
!MESSAGE Lost connection to agent
!STACK 0
java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/java.io.BufferedInputStream.read1(BufferedInputStream.java:290)
	at java.base/java.io.BufferedInputStream.read(BufferedInputStream.java:351)
	at java.base/java.io.FilterInputStream.read(FilterInputStream.java:107)
	at org.eclipse.tcf.core.ChannelTCP.get(ChannelTCP.java:185)
	at org.eclipse.tcf.core.StreamChannel.read(StreamChannel.java:84)
	at org.eclipse.tcf.core.AbstractChannel$1.run(AbstractChannel.java:327)

!ENTRY org.eclipse.cdt.dsf 4 -1 2021-05-05 23:14:15.500
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at jdk.internal.reflect.GeneratedMethodAccessor61.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession.access$2(DsfSession.java:481)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.util.concurrent.RejectedExecutionException: Display org.eclipse.swt.widgets.Display@4f676900 is disposed
	at org.eclipse.cdt.dsf.ui.concurrent.SimpleDisplayExecutor.execute(SimpleDisplayExecutor.java:84)
	at org.eclipse.cdt.dsf.ui.viewmodel.AbstractVMAdapter.handleEvent(AbstractVMAdapter.java:254)
	at org.eclipse.cdt.dsf.ui.viewmodel.datamodel.AbstractDMVMAdapter.eventDispatched(AbstractDMVMAdapter.java:102)
	... 12 more
Caused by: org.eclipse.swt.SWTException: Device is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4720)
	at org.eclipse.swt.SWT.error(SWT.java:4635)
	at org.eclipse.swt.SWT.error(SWT.java:4606)
	at org.eclipse.swt.widgets.Display.error(Display.java:1193)
	at org.eclipse.swt.widgets.Display.asyncExec(Display.java:689)
	at org.eclipse.cdt.dsf.ui.concurrent.SimpleDisplayExecutor.execute(SimpleDisplayExecutor.java:81)
	... 14 more
Root exception:
java.util.concurrent.RejectedExecutionException: Display org.eclipse.swt.widgets.Display@4f676900 is disposed
	at org.eclipse.cdt.dsf.ui.concurrent.SimpleDisplayExecutor.execute(SimpleDisplayExecutor.java:84)
	at org.eclipse.cdt.dsf.ui.viewmodel.AbstractVMAdapter.handleEvent(AbstractVMAdapter.java:254)
	at org.eclipse.cdt.dsf.ui.viewmodel.datamodel.AbstractDMVMAdapter.eventDispatched(AbstractDMVMAdapter.java:102)
	at jdk.internal.reflect.GeneratedMethodAccessor61.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:541)
	at org.eclipse.cdt.dsf.service.DsfSession.access$2(DsfSession.java:481)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:408)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.eclipse.swt.SWTException: Device is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4720)
	at org.eclipse.swt.SWT.error(SWT.java:4635)
	at org.eclipse.swt.SWT.error(SWT.java:4606)
	at org.eclipse.swt.widgets.Display.error(Display.java:1193)
	at org.eclipse.swt.widgets.Display.asyncExec(Display.java:689)
	at org.eclipse.cdt.dsf.ui.concurrent.SimpleDisplayExecutor.execute(SimpleDisplayExecutor.java:81)
	... 14 more

!ENTRY org.eclipse.tcf 4 0 2021-05-05 23:14:15.652
!MESSAGE Failed to shutdown TCF event dispatch thread
!STACK 0
java.lang.InterruptedException
	at java.base/java.lang.Object.wait(Native Method)
	at java.base/java.lang.Thread.join(Thread.java:1305)
	at java.base/java.lang.Thread.join(Thread.java:1379)
	at org.eclipse.tcf.EventQueue.shutdown(EventQueue.java:76)
	at org.eclipse.tcf.Activator.stop(Activator.java:154)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$4.run(BundleContextImpl.java:899)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$4.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.stop(BundleContextImpl.java:891)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.stopWorker0(EquinoxBundle.java:1029)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.stopWorker(EquinoxBundle.java:370)
	at org.eclipse.osgi.container.Module.doStop(Module.java:658)
	at org.eclipse.osgi.container.Module.stop(Module.java:520)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.decStartLevel(ModuleContainer.java:1885)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1760)
	at org.eclipse.osgi.container.SystemModule.stopWorker(SystemModule.java:275)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle$EquinoxSystemModule.stopWorker(EquinoxBundle.java:202)
	at org.eclipse.osgi.container.Module.doStop(Module.java:658)
	at org.eclipse.osgi.container.Module.stop(Module.java:520)
	at org.eclipse.osgi.container.SystemModule.stop(SystemModule.java:207)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle$EquinoxSystemModule$1.run(EquinoxBundle.java:220)
	at java.base/java.lang.Thread.run(Thread.java:834)
!SESSION 2021-05-06 08:41:50.047 -----------------------------------------------
eclipse.buildId=unknown
java.version=11.0.5
java.vendor=Amazon.com Inc.
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY com.silabs.ss.platform.jetty.server 1 0 2021-05-06 08:41:55.989
!MESSAGE ContentServer started at: 127.0.0.1, 50369

!ENTRY com.silabs.ss.platform.rcp.core 1 0 2021-05-06 08:41:56.021
!MESSAGE com.silabs.ss.tool.launcher.product found! Writing main studio shared file

!ENTRY com.silabs.ss.platform.content.core 4 0 2021-05-06 08:42:27.559
!MESSAGE Local timestamp file does not exist: C:\SiliconLabs\SimplicityStudio\v5\offline\common\Simplicity_Studio_Terms_of_Use_and_Privacy_Statement.timestamp

!ENTRY com.silabs.ss.platform.content.core 4 0 2021-05-06 08:42:40.355
!MESSAGE Local timestamp file does not exist: C:\SiliconLabs\SimplicityStudio\v5\offline\common\Simplicity_Studio_Terms_of_Use_and_Privacy_Statement.timestamp

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-06 09:30:18.655
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY org.eclipse.cdt.debug.edc 4 0 2021-05-06 09:30:26.153
!MESSAGE Unknown context [rc.root.000050120677] is reported in suspended event. Make sure TCF agent has reported contextAdded event first.

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-06 10:02:22.522
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-06 10:10:07.728
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-06 10:40:50.775
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2021-05-06 10:43:09.668
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)
